README.md

```diff
@@ -2,19 +2,17 @@
 [![Coverage Status](https://coveralls.io/repos/github/codeforsanjose/findahome/badge.svg?branch=development)](https://coveralls.io/github/codeforsanjose/findahome?branch=development)
 [![Dependency Status](https://gemnasium.com/badges/github.com/codeforsanjose/findahome.svg)](https://gemnasium.com/github.com/codeforsanjose/findahome)
 
-![screenshot](http://i.imgur.com/AaVd6VY.png)
+![screenshot](http://www.yanyinchoy.com/assets/img/findahome_may2017_thumbnail.png)
```

Comment:
I'm fine with where this is hosted but, FYI, imgur stores images for years and is free.